components/sidebar/Sidebar.js

```diff
@@ -2,10 +2,12 @@ import React from 'react';
 import PropTypes from 'prop-types';
 
 import NavMenu from './NavMenu';
+import MainLogo from '../logo/MainLogo';
 
 const Sidebar = ({ list = [] }) => {
     return (
         <div className="sidebar flex flex-column noprint">
+            <MainLogo url="/account" className="nodesktop notablet" />
```

Comment:
The `url` needs to come from a `prop`. I'm fine with having `/account` String as default.

Reply:
Done with https://github.com/ProtonMail/react-components/pull/245/commits/031766ecfdd90dd7be3925312267a89b6c456807